untraced/yksg/src/lib.rs

```diff
@@ -183,17 +177,10 @@ impl StopgapInterpreter {
     pub fn from_frames(v: Vec<IncomingFrame>) -> Self {
         let mut frames = Vec::new();
         for fi in v.into_iter() {
-            let body = &fi.body;
-            let layout = body.layout();
-            let mem = FrameLocals {
-                locals: fi.locals,
-                offsets: body.offsets().clone(),
-                layout: Layout::from_size_align(layout.0, layout.1).unwrap(),
-            };
             let frame = Frame {
-                mem,
+                locals: fi.locals,
                 pc: u32::try_from(fi.bbidx).unwrap(),
-                body: fi.body.clone(),
+                body: Arc::clone(&fi.body),
```

Comment:
Is this just to make clearer that `fi.body` is an `Arc`, or does this do something different?

Reply:
> Is this just to make clearer that fi.body is an Arc

Correct.